functions/engine/BellhopEngine.cs

```diff
@@ -139,24 +139,33 @@ public static void Validate()
         }
     }
 
+    class ErrorObject
+    {
+        public object Type { get; set; }
+        public string Message { get; set; }
+        public Exception Exception { get; set; }
+    }
+
     class ResizeObject
     {
         public string Name { get; }
         public string ResourceGroup { get; }
-        public string Subscription { get; }
+        public string SubscriptionId { get; }
+        public string SubscriptionName { get; }
         public string CurrentState { get; }
         public string TargetState { get; }
         public string ResizeAction { get; }
         public JObject GraphData { get; }
-        public List<KeyValuePair<object, string>> _errors = new List<KeyValuePair<object, string>>();
+        public List<Exception> _errors = new List<Exception>();
```

Comment:
Why are you using an `_` in front of errors? Based on what I'm reading online, `_` would be used for private variables.

Reply:
You are correct, this should be changed to a Private variable. Good catch.

Reply:
Just pushed an update to switch this to a private variable instead of public.